testnet/stacks-node/src/tests/mempool.rs

```diff
@@ -843,15 +827,17 @@ fn mempool_setup_chainstate() {
                 let tx_bytes = make_poison(&contract_sk, 5, 1000, microblock_1, microblock_2);
                 let tx =
                     StacksTransaction::consensus_deserialize(&mut tx_bytes.as_slice()).unwrap();
-                chain_state
+                let e = chain_state
                     .will_admit_mempool_tx(
                         &NULL_BURN_STATE_DB,
                         consensus_hash,
                         block_hash,
                         &tx,
                         tx_bytes.len() as u64,
                     )
-                    .unwrap();
+                    .unwrap_err();
+                eprintln!("Err: {:?}", e);
```

Comment:
This prinkln can go away

Reply:
Its there for consistency with the rest of the test.